### ISSUE TITLE
metric scale

### ISSUE THREAD

**Issue by wolf943134497** [lifuguan/IGGT_official#16]
Hi @lifuguan  Thanks for your great works!

How can IGGT achieve metric scale reconstruction capability, and are there plans to release training/fine-tuning scripts in the future for fine-tuning with metric data? Thanks

**Comment by lifuguan** [comment]
Nope, IGGT currently doesn't support metric-depth estimation since it is inherited from VGGT. And we're not allowed to open-source the training pipeline due to company restrictions, but it should be easy to reproduce based on DUSt3R's training code.

**Comment by wolf943134497** [comment]
got it . Thanks
